substra/sdk/schemas.py

```diff
@@ -146,6 +146,30 @@ class DataSampleSpec(_Spec):
     def is_many(self):
         return self.paths and len(self.paths) > 0
 
+    @pydantic.field_validator("paths")
+    @classmethod
+    def resolve_paths(cls, v: typing.Any) -> typing.Any:
```

Comment:
```suggestion
    def resolve_paths(cls, v: Optional[List[pathlib.Path]]) -> Optional[List[pathlib.Path]]:
```

Reply:
Oh great. And I removed the "optional" in that case, as, if the field is specified, this is not optional anymore to have a path. WDT ?